### PR TITLE
[Android] Replace AndroidPrivateKey with PrivateKey

### PR DESCRIPTION
**Pull request by axinging**
Adapt to https://codereview.chromium.org/1474603004.
BUG= XWALK-6492

### CODE REVIEW

**Comment by axinging** [comment]
@lincsoon @fujunwei @rakuco @wuhengzhi
Please help to review soon.

**Comment by lincsoon** [comment]
LGTM

**Comment by axinging** [comment]
@fujunwei  done

File: runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentsClientBridge.java [hunk 1]
```diff
@@ -84,6 +82,8 @@
 
     private float mPageScaleFactor;
 
+    protected ClientCertLookupTable mLookupTable;
```
Comment (wuhengzhi) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentsClientBridge.java]: Should it be declared in parent class?

Comment (axinging) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentsClientBridge.java]: Parent has no reference to this. So I moved  it here.

Comment (wuhengzhi) [inline runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContentsClientBridge.java]: If it won't be used with other child class, declared it here is fine.

**Comment by axinging** [comment]
@rakuco

**Comment by rakuco** [comment]
thanks

**Pull request merged**